mathjax3-ts/util/BitField.ts

```diff
@@ -21,6 +21,8 @@
  * @author dpvc@mathjax.org (Davide Cervone)
  */
 
+const MAXBIT = 1 << 31;
```

Comment:
Should this be a static element of `BitField`, so it can be accessed in subclasses? Or is `BitField` considered final?

Reply:
I don't think there will be subclasses, but I've moved it to a static property, which makes sense.